mailu/templates/dovecot/deployment.yaml

```diff
@@ -65,6 +65,10 @@ spec:
         - name: dovecot
           image: {{ .Values.imageRegistry }}/{{ .Values.dovecot.image.repository }}:{{ default (include "mailu.version" .) .Values.dovecot.image.tag }}
           imagePullPolicy: {{ .Values.dovecot.image.pullPolicy }}
+          {{- with .Values.dovecot.containerSecurityContext }}
```

Comment:
The idiomatic way in Helm us to call `securityContext` to container's security policy and `podSecurityContext` to pod's security policy.

Reply:
Done, thanks for the feedback!